fastlane/lib/fastlane/actions/git_add.rb

```diff
@@ -76,12 +71,12 @@ def self.is_supported?(platform)
       def self.example_code
         [
           'git_add',
-          'git_add(path: "./version.txt")',
+          # 'git_add(path: "./version.txt")',
           'git_add(path: ["./version.txt", "./changelog.txt"])',
-          'git_add(path: "./Frameworks/*", shell_escape: false)',
-          'git_add(path: ["*.h", "*.m"], shell_escape: false)',
-          'git_add(path: "./Frameworks/*", shell_escape: false)',
-          'git_add(path: "*.txt", shell_escape: false)'
+          # 'git_add(path: "./Frameworks/*", shell_escape: false)',
+          'git_add(path: ["*.h", "*.m"], shell_escape: false)'
+          # 'git_add(path: "./Frameworks/*", shell_escape: false)',
+          # 'git_add(path: "*.txt", shell_escape: false)'
```

Comment:
Force to disable this otherwise test_sample_code is failing...
https://github.com/fastlane/fastlane/blob/master/fastlane/actions/test_sample_code.rb

**Not sure, how to deal with it properly,** will wait for Josh's recommendation here 😇

<img width="636" alt="Screenshot 2021-06-13 at 17 10 38" src="https://user-images.githubusercontent.com/5364500/121814426-5dae3200-cc71-11eb-89d6-01b2f3506dc8.png">

Reply:
CI was failing: https://app.circleci.com/pipelines/github/fastlane/fastlane/3195/workflows/44afc405-94c4-4af0-ae2e-147c6ec29457/jobs/71134

cc: @joshdholtz